scrapy/extensions/feedexport.py

```diff
@@ -57,7 +58,9 @@ def _store_in_thread(self, file):
 @implementer(IFeedStorage)
 class StdoutFeedStorage(object):
 
-    def __init__(self, uri, _stdout=sys.stdout):
+    def __init__(self, uri, _stdout=None):
+        if not _stdout:
+            _stdout = sys.stdout if six.PY2 else sys.stdout.buffer
```

Comment:
@lopuhin Thanks for the comment. I think [the rule of three](https://en.wikipedia.org/wiki/Rule_of_three_\(computer_programming\)) is reasonable. In my opinion, there is no need to create a helper method for now.